c++/third_party/zlib/zlib.bzl

```diff
@@ -0,0 +1,18 @@
+"Root build file for Zlib."
+
+cc_library(
+    name = "zlib",
+    srcs = glob(["*.c"]),
+    hdrs = glob(["*.h"]),
+    includes = ["."],
+    # Workaround for zlib warnings and mac compilation. Some issues were resolved in v1.3, but there are still implicit function declarations.
```

Comment:
I've pulled this file out into `c++/third_party/zlib`, rather than being listed inline in the `WORKSPACE` file. This keeps the root `WORKSPACE` clean; users often reference that file (misguided as it may be) for instructions to install a Bazel dependency in their own project.

Plus, `release-prep.sh` uses `awk` on the root `WORKSPACE`, and escaping these file contents may confuse it. Also enables syntax highlighting, which is nice.